net/vallumd/files/vallumd.conf

```diff
@@ -0,0 +1,5 @@
+config vallumd mqtt
+	option host 127.0.0.1
+	option port 1883
+	list topics blacklist4
+	list topics blacklist6
```

Comment:
Yes, the assumption is very likely true.  But the pre-defined setting is also more likely not what they want and in this case no-op should be better than non-defined-op